### PR TITLE
Closed Loop: added qe coverage for bugzilla 1686916 and 1677282

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
### PR Objective
Added automation coverage for Bugzilla's 1686916 and 1677282

### CODE REVIEW

File: tests/foreman/cli/test_subscription.py [hunk 1]
```diff
@@ -228,3 +228,24 @@ def test_negative_manifest_refresh(self):
...
+        :id: 4861bcbc-785a-436d-98ce-14cfef7d6907
+
+        :expectedresults: subscription list contains the start and end date
+
+        :BZ: 1686916
+
+        :CaseImportance: Medium
+        """
+        self._upload_manifest(self.org['id'])
+        subscription_list = Subscription.list(
```
Comment (vijay8451) [inline tests/foreman/cli/test_subscription.py]:
```suggestion
        subscriptions_list = Subscription.list(
```

File: tests/foreman/cli/test_subscription.py [hunk 2]
```diff
@@ -228,3 +228,24 @@ def test_negative_manifest_refresh(self):
...
+            {'organization-id': self.org['id']},
+            per_page=False,
+        )
+        for column in ['start-date', 'end-date']:
+            self.assertTrue(column in subscription_list[0].keys())
```
Comment (vijay8451) [inline tests/foreman/cli/test_subscription.py]:
```suggestion
            self.assertIn(column, subscription_list[0].keys())
```

File: tests/foreman/ui/test_settings.py [hunk 3]
```diff
@@ -125,6 +126,31 @@ def test_positive_update_restrict_composite_view(session, set_original_property_
...
+    """
+    property_name = 'http_proxy'
+    with session:
+        set_original_property_value(property_name)
+        param_value = gen_url()
+        session.settings.update(
+            'name = {}'.format(property_name),
+            param_value
+        )
+        assert session.settings.read(property_name)
```
Comment (vijay8451) [inline tests/foreman/ui/test_settings.py]: a tearDown requires as this setting does impact to `all outgoing HTTP connections.`.

File: tests/foreman/ui/test_settings.py [hunk 4]
```diff
@@ -125,6 +126,31 @@ def test_positive_update_restrict_composite_view(session, set_original_property_
...
+    :CaseImportance: Medium
+
+    """
+    property_name = 'http_proxy'
+    with session:
+        set_original_property_value(property_name)
+        param_value = gen_url()
+        session.settings.update(
+            'name = {}'.format(property_name),
+            param_value
```
Comment (vijay8451) [inline tests/foreman/ui/test_settings.py]:
```suggestion
            gen_url()
```
Can we directly call as above ?

File: tests/foreman/ui/test_settings.py (hunk 3 above)
Comment (omkarkhatavkar) [reply tests/foreman/ui/test_settings.py]: https://github.com/SatelliteQE/robottelo/pull/7406/files#diff-3c2a24bcd6b2d5a25109128adda04e33R54 fixture will take care of it

File: tests/foreman/cli/test_subscription.py (hunk 1 above)
Comment (omkarkhatavkar) [reply tests/foreman/cli/test_subscription.py]: I think subscription_list is more valid here as the list is not single entity. It bound to have multiple objects  e.g. I can not write like repositories_list or products_list same manner

File: tests/foreman/ui/test_settings.py (hunk 4 above)
Comment (omkarkhatavkar) [reply tests/foreman/ui/test_settings.py]: hmm good call

File: tests/foreman/cli/test_subscription.py (hunk 1 above)
Comment (vijay8451) [reply tests/foreman/cli/test_subscription.py]: hummm ...it's fine then :)

**Comment by omkarkhatavkar** [comment]
@SatelliteQE/quality-engineers Please find some time to review this PR, I know you guys are really busy.

**Pull request merged**